app/controllers/users_controller.rb

```diff
@@ -2,12 +2,16 @@
 class UsersController < ApplicationController
   load_permissions_and_authorize_resource
   before_action :set_user
+
   def index
   end
 
   def show
   end
 
+  def profile
```

Comment:
All hail the king, the profile has returned!

The user `show` action should take care of this is doing.